src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
+                Person Risk:{' '}
+                <code>{personRiskEachFormatted}-in-a-million chance</code>
+              </strong>{' '}
+              <br />
+              First, we calculate that each other person in this area has a{' '}
+              <strong>{personRiskEachFormatted}-in-a-million chance</strong>
+              {'  '} of currently having COVID.
+            </li>
+            <li>
+              <strong>
+                Activity Risk: <code>{activityRiskFormatted} chance</code>
+              </strong>
+              <br /> Next, we calcualte the risk of the activity. Assuming 1
+              person at this activity has COVID, then you would have a{' '}
+              <strong>{activityRiskFormatted} chance</strong> of getting COVID.
+              <b>
+                {activityRisk && activityRisk >= MAX_ACTIVITY_RISK
+                  ? ' (NOTE: We have capped this number at the maximum Activity Risk.)'
+                  : ''}
+                {props.data.distance === 'intimate' &&
+                props.data.duration < intimateDurationFloor
+                  ? ' (NOTE: We have applied a minimum Activity Risk for kissing.)'
+                  : ''}
+              </b>
+            </li>
+            <li>
+              <strong>
+                Number of people:{' '}
+                <code>
+                  {props.data.personCount} {personCountSuffixFormatted}
+                </code>
+              </strong>
+            </li>
+            <li>
+              <strong>
+                Total risk:{' '}
+                <code>
+                  ~{pointsFormatted}-in-a-million ({pointsPercentFormatted})
+                </code>
+              </strong>
+              <br />
+              Finally, we multiply Person Risk, Activity Risk, and the number of
+              people to get the total result of roughly{' '}
+              <strong>
+                {pointsFormatted}-in-a-million ({pointsPercentFormatted})
+              </strong>{' '}
+              chance of getting COVID from this activity with these people{' '}
+              <strong>{frequencyFormatted}</strong>.
+            </li>
+            <li>
+              <strong>
+                Frequency: <code>{frequencyFormatted}</code>
+              </strong>
+              <br />
+              {props.repeatedEvent ? (
+                <>
+                  Since you are seeing these people many times per week, the
+                  result you see is{' '}
+                  <strong>
+                    microCOVIDs <u>per week</u>
+                  </strong>
+                  .
+                </>
+              ) : (
+                <>
+                  Since this is a one-time interaction, the results is shown as{' '}
+                  <strong>
+                    microCOVIDs <u>each time</u> you have this interaction
+                  </strong>
+                  . If you do this activity many times in a week, each time you
```

Comment:
Should probably add:
If you do this activity many times with the same people each week, enter it as {intimate ? partner : housemate} to see what the maximum transmission rate is per week.